src/ocean/math/IEEE.d

```diff
@@ -321,7 +322,7 @@ private:
               tmpval &=0xFFFF_FC00;
               asm { ld tmpval, %fsr; }
             */
-           assert(0, "Not yet supported");
+           throw new SanityException("Not yet supported");
```

Comment:
Shall we simply deprecate these functions? It is weird to keep code that does nothing but throwing.

Reply:
Sure. We could even just remove them because they are literally dead code, no user code can possibly call them.

Reply:
Let's just follow usual procedure, no benefit in making exceptions :) New major should be soon enough for this to not matter.